extensions/amp-form/0.1/amp-form.js

```diff
@@ -531,10 +531,7 @@ export class AmpForm {
          * If the submit was called with an event, then we shouldn't
          * manually submit the form
          */
-        let shouldSubmitFormElement = true;
-        if (event) {
-          shouldSubmitFormElement = false;
-        }
+        let shouldSubmitFormElement = Boolean(!event);
```

Comment:
The `!` operator automatically casts to a boolean, so you can omit the constructor if you're negating it.

Reply:
Awesome, yeah I was thinking I should, but then I think I got pulled into a meeting or something. 😄 Thanks!